<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAppMain</class>
 <widget class="QMainWindow" name="FaceAppMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>690</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D-FAST</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/logos/window_icon.png</normaloff>:/logos/window_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="visToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_ShowVisToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Visualisations Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Show Visualisations Toolbar</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_3D_FAST">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;3D-FAST</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/options.png</normaloff>:/icons/options.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionAbout_Plugins">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/plugins.png</normaloff>:/icons/plugins.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Plugins...</string>
   </property>
  </action>
  <action name="action_Landmarks">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/landmarks.png</normaloff>:/icons/landmarks.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Landmarks</string>
   </property>
  </action>
  <action name="actionLicensing">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/licensing.png</normaloff>:/icons/licensing.png</iconset>
   </property>
   <property name="text">
    <string>Licensing</string>
   </property>
  </action>
  <action name="actionModel_Info">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/model_info.png</normaloff>:/icons/model_info.png</iconset>
   </property>
   <property name="text">
    <string>Model &amp;Info...</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/reset_camera.png</normaloff>:/icons/reset_camera.png</iconset>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionSave_Screenshot">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/screenshot.png</normaloff>:/icons/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>Save Screenshot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>FaceAppMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_ShowVisToolbar</sender>
   <signal>triggered(bool)</signal>
   <receiver>visToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
